src/test/resources/dataset-bags/additional-license-by-text/metadata/dataset.xml

```diff
@@ -9,6 +9,7 @@
         <dc:description>A dataset with some files but no additional license</dc:description>
         <dc:creator>A Creator</dc:creator>
         <ddm:created>2015</ddm:created>
+        <ddm:available>2016</ddm:available>
```

Comment:
no, it says "Element value MUST conform to the dcterms:W3CDTF format." which is specified here: https://www.w3.org/TR/NOTE-datetime

our split-multi-deposit is more strict, demanding a complete date, but ddm itself does not require that